src/panels/LogPanel/ScriptLogPanel.tsx

```diff
@@ -0,0 +1,73 @@
+import { useCallback, useEffect, useState } from 'react';
+import { ActionIcon, Group, Tooltip } from '@mantine/core';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+import { FilterList } from '@/components/FilterList/FilterList';
+import { wordBeginningSubString } from '@/components/FilterList/util';
+import { RefreshIcon } from '@/icons/icons';
+
+import { ScriptLogEntry } from './ScriptLogEntry';
+
+// TODO (anden88 2025-04-17): Remove when lists from OpenSpace don't have the
+// ['1']: value, ['2']: value format
+type FileLines = Record<string, string> | undefined;
+
+export function ScriptLogPanel() {
+  const [scriptLogEntries, setScriptLogEntries] = useState<string[] | undefined>(
+    undefined
+  );
+  const luaApi = useOpenSpaceApi();
+
+  const fetchScriptLogEntries = useCallback(async () => {
+    // eslint-disable-next-line no-template-curly-in-string
+    const fileName = await luaApi?.absPath('${LOGS}/ScriptLog.txt');
+
+    if (!fileName) {
+      return;
+    }
+    const data = (await luaApi?.readFileLines(fileName)) as FileLines;
```

Comment:
Since you will know exactly what these lines look like, how about filtering them away from the output for now?

Reply:
I know we want to make a topic for it eventually, but who knows when that will happen so we might just as well do this simple check now